src/test/features/environment-spec.ts

```diff
@@ -122,6 +122,12 @@ const waitForLoadAndEnvMap =
     };
 
 suite('ModelViewerElementBase with EnvironmentMixin', () => {
+  suiteTeardown(() => {
+    // Reset the renderer once at the end of this spec, to clear out all
+    // of the heavy cached image buffers:
+    ModelViewerElementBase[$resetRenderer]();
```

Comment:
Do we really want to destroy the gl context here? Would it be adequate to dispose of and recreate TextureUtils?

Reply:
As we have discussed before, the GL context isn't especially important. What this does do is clear out [a few internal Three.js caches](https://github.com/mrdoob/three.js/blob/dev/src/renderers/WebGLRenderer.js#L569-L572). It's possible that we could dispose of the artifacts held in these caches more discretely, but I'm not sure how to go about doing that comprehensively.

This is only ever done once in the entire test suite (after `environmennt-spec.ts` is run to completion), so it shouldn't be a huge cost 🤷‍♂